js/interactive-guides/modules/content-manager.js

```diff
@@ -731,6 +731,12 @@ var contentManager = (function() {
         if(instruction){
             var instructionID = stepName + '-instruction-' + currentInstructionIndex;
 
+            //scroll so the recently completed instruction is at the top of the browser window
+            if (($("#"+instructionID).length > 0) && (window.innerWidth <= 1170)) {
```

Comment:
Only comment is that in common-multipane.js we defined:
var twoColumnBreakpoint = 1170;

You may want to use twoColumnBreakpoint instead of 1170 above.  But it's really up to you!

Reply:
I agree. I made the change to use twoColumnBreakpoint. Thanks!